src/apps/Pages/Events/EventItem.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {Card, CardHeader, CardTitle, CardText, CardActions, FlatButton} from 'material-ui';
+
+export default (props) => {
```

Comment:
instead of props take only what you need by: `export default ({event}) => {`

and you can remove `const event = props.event;` below